src/api.js

```diff
@@ -0,0 +1,10 @@
+// The "Public API" for notebooks. This lets notebooks and third-party plugins
+// extend and manipulate the notebook
+
+import { addOutputHandler } from './components/output'
+
+export const io = {
```

Comment:
@mdboom would love to get your thoughts on this. I am thinking it might be wise to avoid the `io` namespace because [socket.IO](https://socket.io/) typically uses it. It's not a _default_ namespacing thing, but given it is common, do you think we should use something else? I guess this is a matter of taste.

Reply:
I'm not plugged into the Javascript ecosystem enough to really know...  Happy to make this whatever we decide.

Reply:
i'm in favor of just using this for now so that we can start experimenting, but not getting attached to `io` and being open to change it after thinking about it and discussing it some. see #350